code/datums/gamemode/dynamic/dynamic_rulesets.dm

```diff
@@ -287,3 +287,13 @@
 		if(!check_enemy_jobs(FALSE))
 			return 0
 	return ..()
+
+/datum/dynamic_ruleset/proc/latejoinprompt(var/mob/user = usr, var/ruleset = null)
```

Comment:
```suggestion
/datum/dynamic_ruleset/proc/latejoinprompt(var/mob/user, var/ruleset)
```
`usr` here is dangerous and bad
`ruleset` should never be null

Reply:
fair and fair